src/main/java/com/clouway/chita/ChitaHttpClient.java

```diff
@@ -95,7 +95,11 @@ public HttpResponse execute(HttpRequest request) {
 
         }
 
-        inputStream = conn.getInputStream();
+        try {
```

Comment:
I think that this should be placed in the HttpResponse class which constructor need to be protected for the package, to disallow external usages.